www/javascript/site-jw-player-media-display.js

```diff
@@ -560,22 +560,24 @@ SiteJwPlayerMediaDisplay.prototype.drawDialogs = function()
 	this.overlay.appendChild(document.createTextNode(''));
 	this.container.appendChild(this.overlay);
 
-	// if the video has been watched before, and we're more than 60s from the
-	// end or start, show a message allowing the viewed to resume or start
-	// from the beginning
-	if (this.start_position > 0) {
+	 // if the video has been watched before, and we're more than
+	 // 10s from the start, or 60s from the end, show a
+	 // message allowing the viewed to resume or start
+	 // from the beginning
+	 if (this.start_position > 0) {
 		this.appendResumeMessage();
 
 		var that = this;
 		this.player.on('ready', function () {
-			if (that.start_position > 60 &&
+			if (that.start_position > 10 &&
 				that.start_position < that.duration - 60) {
 
 				that.displayResumeMessage();
 			}
 		});
 	}
 
+
```

Comment:
This extra line isn't necessary.

Reply:
Heh, I originally removed the line unconsciously, then a diff showed it was there so I put it back. Removed.

Reply:
I see one line removed, but there's still double blank lines on 578 and 59

Reply:
569? I'll remove that.